src/Components/GSignInButton.js

```diff
@@ -49,7 +48,16 @@ const GSignInButton = ({onSignIn}) => {
     }
   };
 
-  _getMnemonic = async () => {
+  _getMnemonic = async uid => {
+    // 1. Read mnemonic from the store
+    /*
```

Comment:
why comment this?

Reply:
I have an error in calling retrieveMnemonic function. Think there is a bug with the last changes for mnemonic storing by uuid. I wrote to Hao about that.

Reply:
@lpetkov-sw I merged Haos PR to master, so you can rebase and then it should work